1es-azure-pipeline.yml

```diff
@@ -0,0 +1,84 @@
+trigger:
```

Comment:
What is this pipeline used for?

Also good to have link to pipeline at top like this https://github.com/dotnet/project-system/blob/34eb57b35962367b71c2a1d79f6c486945586e24/eng/pipelines/pull-request.yml#L3-L4
@MiYanni started the trend.

Reply:
Thanks a convenient pattern, I like it. Thanks for leaving your feedback :)

Regarding this pipeline, it is the internal one which uses the 1es template, while the other one is for public pipelines not using 1es templates.